src/JustSaying/AwsTools/MessageHandling/ISqsQueue.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.SQS;
+using JustSaying.AwsTools.QueueCreation;
+
+namespace JustSaying.AwsTools.MessageHandling
+{
+    public interface ISqsQueue
+    {
+        IAmazonSQS Client { get; }
+        string QueueName { get; }
+        RegionEndpoint Region { get; }
+        Uri Uri { get; }
```

Comment:
`QueueUri`? `QueueEndpoint`?

Reply:
The containing interface already provides that context.